woocommerce-gateway-stripe.php

```diff
@@ -24,7 +24,7 @@
 /**
  * WooCommerce fallback notice.
  *
- * @since 4.1.2
+ * @since 4.1.15
```

Comment:
What is this change for?

Reply:
Oh right this is different than the current version number. I'll change this back.